package com.salesforce.functions.jvm.runtime.sfjavafunction.exception;

public class FunctionThrewExceptionException extends SalesforceFunctionException {
    public FunctionThrewExceptionException(Throwable cause) {
        super(cause);
    }
}
